components/account/AccountTabs.vue

```diff
@@ -34,6 +36,7 @@ const tabs = $computed<CommonRouteTabOption[]>(() => [
     },
     display: t('tab.media'),
     icon: 'i-ri:camera-2-line',
+    dataGlean: 'profile.tabs.media',
```

Comment:
Same here with these events - I don't see the engagement events fire in console or ping debugger, I only see the `link_click` and `page_view` events